### PR TITLE
Fixed metatype description

### PR DESCRIPTION
**Pull request by utle**

### CODE REVIEW

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -84,8 +84,8 @@ loginFormURL.desc=Specifies the global URL of a form login page including the ro
...
 
-allowAuthenticationFailOverToAuthMethod=Failover method when certificate login fails
-allowAuthenticationFailOverToAuthMethod.desc=Specifies the authentication fail over method that will be used when the certificate authentication fails or if the certificate is missing. Valid values are BASIC, FORM and APP_DEFINED. APP_DEFINED is only valid when overrideHttpAuthMethod attribute is set to CLIENT_CERT. When APP_DEFINED is set, the authentication method which is  configured in the application is used.
+allowAuthenticationFailOverToAuthMethod=Failover to the authentication method when certificate login fails
+allowAuthenticationFailOverToAuthMethod.desc=Specifies the authentication fail over method that will be used when the certificate authentication fails or if the certificate is missing. Valid values are BASIC, FORM and APP_DEFINED. APP_DEFINED is only valid when overrideHttpAuthMethod attribute is set to CLIENT_CERT. When APP_DEFINED is set, the authentication method which is configured in the application is used.
```
Comment (helyarp) [inline dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties:88]: - Change "fail over" to "failover"
- Change "will be used" to "is used"
- Add a comma after FORM
- Change "method which" to "method that"

**Review by helyarp — COMMENTED** [review]
Suggested changes

**Comment by utle** [comment]
Hi Pam,
I am using https://github.com/OpenLiberty/open-liberty/pull/22228 instead.

File: dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (utle) [reply dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties:88]: Updated in the other PR

**Pull request closed without merging**